porcupine/plugins/pluginmanager.py

```diff
@@ -56,12 +56,12 @@ def __init__(self, master: tkinter.Misc) -> None:
         self._plz_restart_label = ttk.Label(self.content_frame)
         self._plz_restart_label.pack(side="bottom", fill="x")
 
-        column_sizes = [120, 150, 180]
+        column_sizes = (120, 150, 180)
 
-        left_side = ttk.Frame(panedwindow)
+        left_side = ttk.Frame(panedwindow, padding=10)
```

Comment:
The left side of the plugin manager looks a bit ridiculous IMO with that much padding. UIs with too much padding make me feel like I am struggling to see the content of a tiny box.

```suggestion
        left_side = ttk.Frame(panedwindow, padding=5)
```

Reply:
Yeah, it looks indeed better with just 5 px